uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -27,14 +27,14 @@ define([
       var absenceTypesAndIds,
         initialLeaveRequestAttributes = {}, //used to compare the change in leaverequest in edit mode
         mode = '', //can be edit, create, view
-        role = '', //could be manager, owner or admin
+        role = $rootScope.isManager ? 'manager' : 'owner', //could be manager, owner or admin
```

Comment:
When we have a way to know how admin can be recognised, we can set another variable like this, and using that same variable request-ctrl can decide, which role it is.

Also added `initUserRole` where the code can be placed in future.